src/pages/Results/Results.js

```diff
@@ -16,39 +17,48 @@ const Results = () => {
     const param = useParams();
     const [text, setText] = useState(decodeURIComponent(param.userText));
     const [checkedItems, setCheckedItems] = useState({});
+    const [SearchSelected, setSearchResult] = useState(true);
```

Comment:
i think it would be good to rename `setSearchResult` to something else, the name is kind of confusing because it implies that the results themselves are going to be altered when they are not. maybe `setSearchSelected`?